DotNetHowToIndexers/Program.cs

```diff
@@ -1,16 +1,19 @@
 ﻿using System;
+using System.Net;
+using System.Threading.Tasks;
 using Microsoft.Azure.Search;
 using Microsoft.Azure.Search.Models;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Rest.Azure;
 
 namespace AzureSearch.SDKHowTo
 {
     /// <summary>
     /// Demo of Azure Search indexer for Azure SQL
     /// </summary>
-    sealed class Program
+    public sealed class Program
     {
-        static void Main(string[] args)
+        public static async Task Main(string[] args)
```

Comment:
Executable entry point methods cannot be async (AFAIK) -  I think this won't even compile... (because no entry points would be found), but even if it will compile as the entry point for the exe, the process would exit at the first 'await' call and terminate as there is no "caller" that would wait for it top complete. Am I missing something?